akvo/rsr/spa/app/root.jsx

```diff
@@ -18,9 +18,11 @@ import IATI from './modules/iati/iati'
 import Profile from './modules/profile/profile'
 
 if (!(env && env.LOCALDEV)) {
+  const hostname = window.location.hostname
   Sentry.init({
     dsn: 'https://5ca590f001844d1493e459a38e3d75f3@sentry.io/218288',
-    release: 'rsr-live@test-release'
+    release: `rsr@${env.RELEASE}`,
+    environment: (hostname === 'rsr.akvo.org' || hostname.endsWith('akvoapp.org')) ? 'prod' : 'test'
```

Comment:
This logic brings back bad memories from my old me.

Both the environment and the SHA are available (or could) as part of the Nginx env in k8s). Is there a reasonable option to use those?

We are already doing some "mangling" of files at the start up of the nginx process at https://github.com/akvo/akvo-rsr/blob/master/nginx/start.sh. Not that I like this mangling at all but maybe you can think of a better approach.

Otherwise, LGTM.